src/main/java/core/basesyntax/supliers/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.supliers;
+
+import core.basesyntax.interfaces.Color;
+import core.basesyntax.shapes.Circle;
+import core.basesyntax.shapes.IsoscelesTrapezoid;
+import core.basesyntax.shapes.Rectangle;
+import core.basesyntax.shapes.RightTriangle;
+import core.basesyntax.shapes.Shape;
+import core.basesyntax.shapes.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM_RADIUS = 10;
+    private static final int RANDOM_INT_BOUND = 7;
+    private static final int TOTAL_SHAPES = 5;
+
+    public Shape getRandomFigure() {
+        Random random = new Random();
+        String randomColor = new ColorSupplier().getRandomColor();
```

Comment:
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked
same for `ColorSupplier`